pixy/httpapi.go

```diff
@@ -210,6 +289,16 @@ type consumeHTTPResponse struct {
 	Offset    int64  `json:"offset"`
 }
 
+type partitionOffsetView struct {
```

Comment:
This has exactly the same structure as `partitionOffset` in `admin.go`. Why not just add JSON tags to that struct? It would also save you from needing to copy data from one to basically the same struct in `handleGetOffsets()`.